README.md

```diff
@@ -1,9 +1,9 @@
-This project contains the static files used to display a site maintenance page.
+# RAS-maintenance pages
 
-The project contains a Staticfile, this allows the project to be cf pushed to Cloud Foundry where it will run in an nginx static buildpack container.
+This project contains the static files used to display a site maintenance page. It uses a variable called `UNPLANNED_MAINTENANCE ` which determines which 
+maintenance page is shown.
 
-
-#Blue green deployment
+# Blue green deployment
```

Comment:
I don't think this section is relevant, probably can be removed. I never used it when messing about with devtest, I just pushed the app and used it on there

Reply:
Just spoke to Gav now and we're gonna keep this bit in for the time being. The deploy shell script it mentions could actually be really helpful for deploying the maintenance page on prod. Instead of doing it manually we could eventually use this script if we tinker with it

Reply:
👍